templates/advantage/index.html

```diff
@@ -44,6 +46,15 @@ <h2>Your {% if subscriptions['total_subscriptions'] > 1 %}{{subscriptions['total
         </div>
       </div>
       {% if subscriptions['has_monthly'] %}
+        {% if payment_method_warning %}
+          <div class="row">
+            <div class="p-notification--caution">
+              <p class="p-notification__response" role="status">
+                <span class="p-notification__status">Payment method:</span>You need to <a href="/advantage/payment-methods{% if is_test_backend %}?test_backend=true{% endif %}">update your payment methods</a> to ensure there is no interruption to your Ubuntu Advantage subscriptions
```

Comment:
Technically, it might happen that 3DS got stuck as well. In that case, the payment method would be fine, it's just that we need to authorize it.

This would happen if the user closes (or otherwise loses) the session right when they were supposed to authorize the purchase, and then they come back and want to resume the purchase. Or when a monthly renewal requires 3DS for some reason (should be rare, but it's theoretically possible).

Also, when a purchase is stuck due to a missing payment method, the corrective action is to ask for a new card, set it (SetStripeCustomerInfo) and then retry the payment of that purchase's invoice.

All of that being said, I think this could be better phrased as "There is a pending purchase on your account that requires further input from you. Click here to resolve it."

Reply:
I don't agree - the wording is too complex for what is a very rare edge case. Please keep the copy as-is.